nle/nethack/nethack.py

```diff
@@ -100,7 +100,12 @@ def __init__(
             )
 
         # Create a HACKDIR for us.
-        self._vardir = tempfile.mkdtemp(prefix="nle")
+        self._tempdir = tempfile.TemporaryDirectory(prefix="nle")
+        self._vardir = self._tempdir.name
```

Comment:
tempdir._name might return AttributeError, not sure if we want to check that somehow. https://hg.python.org/cpython/file/3.6/Lib/tempfile.py#l725

Reply:
I think you might be confusing this with `TemporaryFile`: https://hg.python.org/cpython/file/3.6/Lib/tempfile.py#l790